packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -8,8 +9,7 @@ import { wireService, wireServiceWithoutPayload } from '../../wiring/wires.facto
 import { filter, mapWire } from '../../wiring/wires.operators';
 import { Wire } from '../../wiring/wiring.types';
 import { createWiring } from '../../wiring/wiring.utils';
-import { DefaultSessionService } from './service/session.service';
-import { DefaultPDPAddToCartService } from './service/index';
+import { DefaultPDPAddToCartService } from './service';
```

Comment:
Avoid importing from barrel file

```suggestion
import { DefaultPDPAddToCartService } from './service/pdp-add-to-cart.service';

```